src/ts/ConvivaAnalytics.ts

```diff
@@ -463,6 +467,19 @@ export class ConvivaAnalytics {
     if (this.lastSeenBitrate) {
       this.convivaVideoAnalytics.reportPlaybackMetric(Conviva.Constants.Playback.BITRATE, this.lastSeenBitrate);
     }
+
+    // Send the session init audio language values.
+    this.updateAudioTrack(this.player.getAudio());
+
+    // Check if at session init has a subtitle enabled.
+    const enableSubtitle = this.player.subtitles.list().filter((i) => i.enabled);
```

Comment:
Please note that `this.player.subtitles` may be undefined if no subtitle modules are added to the modular player. Please make sure to cover this case as otherwise the code would run into an exception in this scenario.

Reply:
Thanks for your feedback, I just push commit with the fix.